content/docs/hooks-custom.md

```diff
@@ -249,4 +248,4 @@ function Todos() {
 }
 ```
 
-The need to manage local state with a reducer in a complex component is common enough that we've built the `useReducer` Hook right into React. You'll find it together with other built-in Hooks in the [Hooks API reference](/docs/hooks-reference.html).
+La necesidad de gestionar estados locales con un reductor en un componente complejo es lo suficientemente común que hemos construido el Hook `useReducer` justo dentro de React. Vas a encontrarlo junto con otros Hooks incororados en la[referencia a la API de Hooks](/docs/hooks-reference.html).
```

Comment:
```suggestion
La necesidad de gestionar estados locales con un reductor en un componente complejo es lo suficientemente común que hemos construido el Hook `useReducer` justo dentro de React. Vas a encontrarlo junto con otros Hooks incorporados en la [referencia a la API de Hooks](/docs/hooks-reference.html).
```

Reply:
Isn't `justo dentro de React` too much of a literal translation? 
Maybe leaving `dentro de React` is enough, leaving `justo` out.

Finally, for the last sentence, I suggest starting it this way:
> Lo encontrarás junto con otros Hooks ...